src/runtime/agas/addressing_service.cpp

```diff
@@ -1731,51 +1730,11 @@ future<hpx::id_type> addressing_service::on_symbol_namespace_event(
         ));
 }
 
-}}
-
-///////////////////////////////////////////////////////////////////////////////
-typedef hpx::agas::server::symbol_namespace::on_event_action
-    symbol_namespace_on_event_action;
-
-HPX_REGISTER_BROADCAST_ACTION_DECLARATION(symbol_namespace_on_event_action,
-    symbol_namespace_on_event_action)
-HPX_REGISTER_BROADCAST_ACTION_ID(symbol_namespace_on_event_action,
-    symbol_namespace_on_event_action,
-    hpx::actions::broadcast_symbol_namespace_on_event_action_id)
-
-namespace hpx { namespace agas
-{
-    namespace detail
-    {
-        std::vector<hpx::id_type> find_all_symbol_namespace_services()
-        {
-            std::vector<hpx::id_type> ids;
-            for (hpx::id_type const& id : hpx::find_all_localities())
-            {
-                ids.push_back(hpx::id_type(
-                    agas::symbol_namespace::get_service_instance(id),
-                    id_type::unmanaged));
-            }
-            return ids;
-        }
-    }
-
 /// Invoke the supplied hpx::function for every registered global name
```

Comment:
This comment should be fixed. There is no function involved there anymore.

Reply:
Good catch, will fix.